rig-core/src/one_or_many.rs

```diff
@@ -27,6 +27,22 @@ impl<T: Clone> OneOrMany<T> {
         self.rest.clone()
     }
 
+    /// After `OneOrMany<T>` is created, add an item of type T to the `rest`.
+    pub fn add(&mut self, item: T) {
+        self.rest.push(item);
+    }
```

Comment:
We should follow the same method naming convention as the `std::vec::Vec`, so `push` instead of `add` (see https://doc.rust-lang.org/std/vec/struct.Vec.html#method.push)